WalletWasabi/CoinJoin/Coordinator/Rounds/CoordinatorRound.cs

```diff
@@ -1061,6 +1061,22 @@ public IEnumerable<Alice> GetAlicesBy(AliceState state)
 			}
 		}
 
+		public IEnumerable<Alice> GetAlicesBy(BitcoinAddress changeOutputAddress)
+		{
+			using (RoundSynchronizerLock.Lock())
+			{
+				return Alices.Where(x => x.ChangeOutputAddress == changeOutputAddress).ToList();
+			}
+		}
+
+		public IEnumerable<Bob> GetBobsBy(BitcoinAddress activeOutputAddress)
```

Comment:
This and all the callers can be removed, because of https://github.com/nopara73/WalletWasabi/blob/5f6bc59bfb59814e0eaf439209b36599fe54266d/WalletWasabi/Extensions/NBitcoinExtensions.cs#L188